content/user/reports/OrgStats.md

```diff
@@ -21,12 +21,22 @@ To take a closer look at this information, select the **View** link under each h
 
 ## Licenses
 
-The Licenses tab features an Activity trend chart of data on license consumption over the past 12 months. Hover over a data point on the Assessment or Protection trend lines to see how many licenses were used that month. The thermometer chart below shows the total number of licenses purchased compared to the number being used. The timeline shows how many licenses are about to expire on given dates. For a different view of the data, take a look at the circular charts with breakdowns by fraction and percentage for Assessment and Protection. 
+The Licenses tab features an activity trend chart of data on license consumption over the past year. Hover over a data point on the Assess or Protect trend lines to see how many licenses were used each month. The thermometer chart below shows the total number of licenses purchased compared to the number being used. The timeline shows how many licenses are about to expire on given dates. For a different view of the data, the circular charts show breakdowns by fraction and percentage for Assess and Protect. 
 
-If your organization doesn't own any licenses for either Protection or Assessment, Contrast alerts you to the count of unlicensed assets in that mode.
+>**Note:** If your organization doesn't own any Protect or Assess licenses, Contrast alerts you to the count of unlicensed assets in that mode.
 
 <a href="assets/images/Org-stats-licenses.png" rel="lightbox" title="Organization statistics for licenses"><img class="thumbnail" src="assets/images/Org-stats-licenses.png"/></a>
 
+### Protect view 
+
+Take a closer look at your Protect license usage by clicking the chart beneath the Protect trend line. In the Protect Usage view, data for the current month is displayed in a trend chart as well as a quick view of **Usage Statistics**. Hover over data points in the trend chart to see the number of Protect servers used and the number of available licenses that remained for each day. Use the dropdown menu above the chart to view data from a different month within the past year. 
```

Comment:
Just some suggested wording - feel free to use what makes sense to you: "Take a closer look at how you use your Protect licenses by clicking within the Protect area of the trend chart. This will switch you over to Protect Usage mode, where you can view data for the current month in a visualization as well as in a summary panel of Usage Statistics. Hover over data points in the trend chart to see both the number of Protect servers used, and the number of available licenses that remained for each day of the selected month. Used the dropdown menu above the chart to view data from any previous month within the past year."